simplemma/__init__.py

```diff
@@ -7,8 +7,9 @@
 __version__ = "0.9.1"
 
 
-from .language_detector import in_target_language, lang_detector
+from .language_detector import in_target_language, langdetect
```

Comment:
The new name can stay but why not.

Reply:
As you want, I thought that for new names, users can use the class and for old names, they can use the old methods.

Reply:
OK, let's leave it that way.